src/Features/Core/Portable/CodeRefactoringHelpers.cs

```diff
@@ -218,7 +141,7 @@ private static async Task<TextSpan> GetExpandedTextSpan(Document document, TextS
         /// Returns unchanged <paramref name="span"/> in case <see cref="TextSpan.IsEmpty"/>.
         /// Returns empty Span with original <see cref="TextSpan.Start"/> in case it contains only whitespace.
         /// </remarks>
-        private static async Task<TextSpan> GetStrippedTextSpan(Document document, TextSpan span, CancellationToken cancellationToken)
+        public static async Task<TextSpan> GetStrippedTextSpan(Document document, TextSpan span, CancellationToken cancellationToken)
```

Comment:
Nit: I think we sholud rename this to GetTrimmedTextSpan.  'Trim' is a well known .net term with strings to indicate (by default) that whitespace should be removed.